xdcs-server/src/main/java/pl/edu/agh/xdcs/db/entity/AgentEntity.java

```diff
@@ -15,11 +17,16 @@
 @Setter
 @NoArgsConstructor
 @Entity(name = "Agent")
-@Table(name = "XDCS_AGENT_")
+@Table(name = "XDCS_AGENT_", indexes = {
+        @Index(columnList = "NAME_")
+})
 public class AgentEntity extends BaseEntity {
     @Column(name = "NAME_")
     private String name;
 
     @Column(name = "DISPLAY_NAME_")
     private String displayName;
+
+    @Column(name = "STATUS_")
+    private Agent.Status status;
```

Comment:
jak persystujesz status to on powinien być z paczki `db`, więc albo ten przenosisz tutaj albo robisz nowy

Reply:
(proponuję przenieść)